.appveyor.yml

```diff
@@ -1,80 +1,54 @@
-version: 0.0.1-branch-{branch}-build-{build}
+version: 2.3.18-branch-{branch}-build-{build}
+image: Visual Studio 2015
 cache:
-    - c:\protobuf
-    - c:\protoc
-    - c:\zlib
+    - c:\protobuf-release
+    - c:\zlib-release
 environment:
     matrix:
-        - vc_arch: amd64
-          choco_arch:
-          nuget_arch: x64
+        - qt_ver: 5.9\msvc2015_64
+          protobuf_ver: 3.4.1
+          zlib_ver: 1.2.11
+          cmake_generator: Visual Studio 14 2015 Win64
+          cmake_toolset: v140,host=x64
           target_arch: x86_64
-          qt_ver: 5.6\msvc2013_64
-          bintray_path: Win64
-          MYSQL_DRIVER_URL: https://dev.mysql.com/get/Downloads/Connector-C/mysql-connector-c-6.1.6-winx64.zip
-          MYSQL_DRIVER_ARCHIVE: mysql-connector-c-6.1.6-winx64.zip
-          MYSQL_DRIVER_NAME: mysql-connector-c-6.1.6-winx64
-        - vc_arch: amd64_x86  # cross-compile from amd64 to x86
-          choco_arch: --x86
-          nuget_arch: Win32
+        - qt_ver: 5.6\msvc2015 # last version compatible with Windows XP, Vista
+          protobuf_ver: 3.4.1
```

Comment:
Travis still uses protobuf 2 instead?

Reply:
Doesn't really matter, we're still on p2 syntax

Reply:
I know. I was just aiming for consistency between our ci build environments...